RadixWallet/Clients/GatewayAPI/GatewayAPIClient/GatewayAPIClient+Live.swift

```diff
@@ -70,7 +70,7 @@ extension GatewayAPIClient {
 				urlRequest.timeoutInterval = timeoutInterval
 			}
 
-			let data = try await httpClient.executeRequest(urlRequest)
+			let data = try await httpClient.executeRequest(urlRequest, nil)
```

Comment:
We can define a helper method to avoid sending nil most of the times
```Swift
extension HTTPClient {
	func executeRequest(_ request: URLRequest) async throws -> Data {
		try await executeRequest(request, nil)
	}
}
```

or if you follow my previous comment:
```Swift
extension HTTPClient {
	func executeRequest(_ request: URLRequest) async throws -> Data {
		try await executeRequest(request, [.ok])
	}
}
```